src/btree/bt_split.c

```diff
@@ -1803,7 +1803,7 @@ __split_insert(WT_SESSION_IMPL *session, WT_REF *ref)
     child->home = ref->home;
     child->pindex_hint = ref->pindex_hint;
     F_SET(child, WT_REF_FLAG_LEAF);
-    child->state = WT_REF_MEM; /* Visible as soon as the split completes. */
```

Comment:
Stores should be `WT_REF_SET_STATE`, I can understand why it isn't but its strange having this hanging out here by itself.